profile/CSIP.xml

```diff
@@ -759,10 +759,10 @@
             <requirement ID="CSIP60" REQLEVEL="MUST" RELATEDMAT="VocabularyFileGrpAndStructMapDivisionLabel" EXAMPLES="fileSecExample1 fileSecExample2">
                   <description>
                     <head>File grouping</head>
-                      <p xmlns="http://www.w3.org/1999/xhtml">There are one or more file group (fileGrp) elements present grouping the files in the main catagorisation of: Documentation, Schemas and Representations.</p>
-                      <p xmlns="http://www.w3.org/1999/xhtml">These main file groups described are present even if there are no files referenced in the group to maintain the logical structure of the package.</p>
-                      <p xmlns="http://www.w3.org/1999/xhtml">For each file group there are additional elements used which are described in their own requriments.</p>
-                      <p xmlns="http://www.w3.org/1999/xhtml">Its possible to add other user defined file groups which then will use the same additional requriments.</p>
+                      <p xmlns="http://www.w3.org/1999/xhtml">There are one or more file group (fileGrp) elements present, which divide the files into categories: Documentation, Schemas and Representations.</p>
```

Comment:
There is a conflict here:
>There are one or more file group (fileGrp) elements present

vs line 763:
>The file groups MUST be present even if there are no referenced files

If some file groups are mandatory, there should be an exhaustive list of them, e.g.:

"The fileSec MUST have one or more file group (fileGrp) elements, which divide the files into categories. To maintain the logical structure of the package, file groups for the categories "Documentation," "Schemas" and "Representations" MUST be present even if there are no files referenced in them."

Reply:
[VocabularyFileGrpAndStructMapDivisionLabel](http://earkcsip.dilcis.eu/#VocabularyFileGrpAndStructMapDivisionLabel) is about these categories - maybe the whole vocabulary should have mandatory file groups? Currently, there are the three above mentioned values and "Metadata."

Reply:
No proposing getting rid of CSIP60 altogether as it's kind of redundant.

Reply:
I feel I lack a strong enough grasp of the whole picture to take position on this. After splitting `fileSec/fileGrp` into "Documentation," "Schemas" and "Representations," a generic `fileSec/fileGrp` requirement seems unnecessary, but there might still be some useful general things about `fileSec/fileGrp` that are best stated in a separate requirement. E.g. regarding structuring: nesting file groups to mirror the folder structure vs presenting them in a flat list. So I'd add it to the todo list of the dedicated task force I suggested in #426.